src/Shared/Utils/Configuration/ConfigFile.cs

```diff
@@ -6,7 +6,7 @@ public class ConfigFile
     {
         public StructureJson structure = new();
         public Language language = new();
-        public int version = 1;
+        public int version = 2;
```

Comment:
Now that I think about it, a Major.Minor versioning scheme seems more reasonable.
```suggestion
        public string version = "1.1";
```